modules/cert-manager-acme-dns-providers.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc
+
+:_content-type: CONCEPT
+[id="cert-manager-acme-dns-providers_{context}"]
+= Supported DNS-01 providers
+
+The {cert-manager-operator} supports the following DNS-01 providers for ACME issuers:
+
+* Amazon Route 53
+* Azure DNS
++
+[NOTE]
+====
+{cert-manager-operator} does not support using Azure Active Directory (Azure AD) pod identities to assign a managed identity to a pod.
+====
+* Google Cloud DNS
++
+[NOTE]
+====
+{cert-manager-operator} does not support using Google Kubernetes Engine (GKE) Workload Identity.
```

Comment:
@bergerhoffer , "Google Kubernetes Engine (GKE)" is a concept which means Google's k8s cluster. "Workload Identity" is another concept which means Google cloud's authentication feature. Any k8s cluster can enable "Workload Identity", including OCP cluster. Thus, `using Google Kubernetes Engine (GKE)` is unexpected wording. [OCPBUGS-9998](https://issues.redhat.com/browse/OCPBUGS-9998) is filed to let RH's cert-manager operator support "Workload Identity" when OCP enables "Workload Identity". Below is suggestion
```suggestion
{cert-manager-operator} does not support using Google workload identity federation in Managed state.
```

Reply:
@xingxingxia What does the "in Managed state" mean? When the Operator is in the `Managed` state in OCP?

Also, is it okay to capitalize "Workload Identity" still? Per the Google docs, it looks to be capitalized

Reply:
It works when "managementState: Managed" is changed to "managementState: Unmanaged". But "managementState: Unmanaged" is not "supported" officially, because "managementState: Unmanaged" means the operator will not operate the operand. Your current wording "The cert-manager Operator for Red Hat OpenShift does not support using Google Workload Identity federation." is OK, only less full of meanings.

Reply:
Thanks, I'll investigate the capitalization a bit more today and make an adjustment if necessary

Reply:
I'm seeing it capitalized some places in the Google docs, and lowercase others. I'd personally prefer it to be lowercase, so I'll update to use that.